app/views/hyrax/collections/show.html.erb

```diff
@@ -1,73 +1,89 @@
-    <div class="col-md-12">
-
-      <% unless @presenter.banner_file.blank? %>
-          <div class="hyc-banner" style="background-image:url(<%= @presenter.banner_file %>)">
-      <% else %>
-          <div class="hyc-generic">
-      <% end %>
-
-      <div class="hyc-title">
-        <h1><%= @presenter.title.first %></h1>
-        <%= @presenter.collection_type_badge %>
-        <%= @presenter.permission_badge %>
+    <%# Title, view, and description %>
+    <div class="hyc-generic">
+      <div class="col-md-8">
+        <div class="hyc-title">
+          <h1><%= @presenter.title.first %></h1>
+          <span class="lg-border-right item-count">
+            <%= @presenter.total_viewable_items %>
+            <%= 'Item'.pluralize(@presenter.total_viewable_items) %>
+          </span>
+          <span>
+            <%= link_to 'View all', Rails.application.routes.url_helpers.search_catalog_path %>
+          </span>
+        </div>
+        <div class="hyc-desc mw-lg">
+          <%= render 'collection_description', presenter: @presenter %>
+        </div>
       </div>
+      <%# Institutional image %>
+      <div class="col-md-4 text-right">
+        <%= image_tag controller.institution_image_path, alt: '' %>
+      </div>
+      <%# Social media share icons %>
+      <div class="col-md-12 text-right share-links">
+        <div class=" lg-border-bottom">
+        <%= render 'hyrax/base/social_media' %>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
 
-      <% unless @presenter.logo_record.blank? %>
-          <div class="hyc-logos">
-            <% @presenter.logo_record.each_with_index  do |lr, i| %>
-
-                <% if lr[:linkurl].blank? %>
-                    <img alt="<%= lr[:alttext] %>" src="<%= lr[:file_location] %>" />
-                <% else %>
-                    <a href="<%= lr[:linkurl] %>">
-                      <img alt="<%= lr[:alttext] %>" src="<%= lr[:file_location] %>" />
-                    </a>
-                <% end %>
-
-            <% end %>
-          </div>
-      <% end %>
-
-      <% unless @presenter.total_viewable_items.blank? %>
-          <div class="hyc-bugs">
-            <div class="hyc-item-count">
-              <b><%= @presenter.total_viewable_items %></b>
-              <%= 'Item'.pluralize(@presenter.total_viewable_items) %></div>
 
-            <% unless @presenter.creator.blank? %>
-                <div class="hyc-created-by">Created by: <%= @presenter.creator.first %></div>
-            <% end %>
 
-            <% unless @presenter.modified_date.blank? %>
-                <div class="hyc-last-updated">Last Updated: <%= @presenter.modified_date %></div>
-            <% end %>
-          </div>
+<div class="hyc-container" itemscope itemtype="http://schema.org/CollectionPage">
+  <div class="row">
+    <div class="hyc-blacklight hyc-bl-results">
+      <div class="row">
+      <div class="col-md-2"></div>
+      <% @member_docs[0,4].each_with_index do |doc, i| %>
```

Comment:
Definitely doesn't need to happen here, but maybe we can rework `@member_docs` to take into account the `thumbnail` metadata field. It's looking like `thumbnail` can be used for the collection "feature works" functionality, and if we can make `@member_docs` aware of that, we can make the first N works always be from `thumbnail`. That would give us a pretty consistent way to access featured works or first N works.